# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: uriloader/exthandler/tests/mochitest/browser.ini
# Commit: f0094c9184c4
# Full Hash: f0094c9184c41675f11efc2d4332ec1a63e570bc
# Author: Gijs Kruitbosch <gijskruitbosch@gmail.com>
# Date: 2019-11-27 09:44:05
# Regressor Bug: 1496380
# File Overlap Count: 1
# Description:
#   Bug 1496380 - stop recursion via the external protocol handler if Firefox is either the default OS handler or a configured external handler, r=mossop
#   
#   This is an initial implementation of this idea that works on mac.
#   I've added a Windows implementation in another commit in this stack. I'll
#   look at a Linux one in a follow-up bug. I do not think we need them in the
# ==============================================================================

diff -r 9dbf3a2c8c37 -r f0094c9184c4 uriloader/exthandler/tests/mochitest/browser.ini
--- a/uriloader/exthandler/tests/mochitest/browser.ini	Tue Nov 26 18:47:20 2019 +0000
+++ b/uriloader/exthandler/tests/mochitest/browser.ini	Tue Nov 26 18:51:04 2019 +0000
@@ -11,5 +11,7 @@
   download.sjs
 [browser_download_always_ask_preferred_app.js]
 [browser_download_privatebrowsing.js]
+[browser_protocolhandler_loop.js]
+skip-if = fission # Bug 1597154
 [browser_remember_download_option.js]
 [browser_web_protocol_handlers.js]